pkg/services/ngalert/api/tooling/definitions/cortex-ruler.go

```diff
@@ -477,6 +477,18 @@ type AlertRuleNotificationSettings struct {
 	MuteTimeIntervals []string `json:"mute_time_intervals,omitempty"`
 }
 
+// swagger:model
+type Record struct {
```

Comment:
I think my PR also introduced this type in this file - but your version is better because of the doc comments and examples. I think you'll need to delete mine? It's called the same thing.